cleanlab/datalab/internal/issue_finder.py

```diff
@@ -119,6 +119,20 @@ def find_issues(
             evenly distributed across the rows.
             The non-zero entries must be the distances between the corresponding examples. Self-distances must be omitted
             (i.e. the diagonal must be all zeros and the k nearest neighbors of each example must not include itself).
+            Within each row, the non-zero entries (distances) must be sorted in ascending order, and the CSR column indices should reflect this order of distances. This sorting ensures correct neighbor relationships for subsequent analyses.
```

Comment:
I think the entire docstring here should just refer reader to the docstring of `knn_graph` argument in `Datalab.find_issues()` with link. No need to repeat the full docstring which is huge

Reply:
Done.